compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/parser/BLangNodeTransformer.java

```diff
@@ -4633,6 +4647,16 @@ private BLangIdentifier createIdentifier(Location pos, String value, Set<Whitesp
         return bLIdentifer;
     }
 
+    private BLangLiteral createEmptyStringLiteral(Location pos) {
+        BLangLiteral bLiteral = (BLangLiteral) TreeBuilder.createLiteralExpression();
+        bLiteral.pos = pos;
+        bLiteral.type = symTable.stringType;
+        bLiteral.type.tag = TypeTags.STRING;
```

Comment:
It is not needed to set the tag here. `symTable.stringType` contains the tag. Please check it.